src/main/js/core/Shape/shapeDefinitions.js

```diff
@@ -291,7 +291,7 @@ export const X_LIGHT = {
                 "M48 12L36 0 24 12 12 0 0 12l12 12L0 36l12 12 12-12 12 12 12-12-12-12 12-12zM36 43.8l-9.9-9.9-2.1-2.1-2.1 2.1-9.9 9.9L4.2 36l9.9-9.9 2.1-2.1-2.1-2.1L4.2 12 12 4.2l9.9 9.9 2.1 2.1 2.1-2.1L36 4.2l7.8 7.8-9.9 9.9-2.1 2.1 2.1 2.1 9.9 9.9-7.8 7.8z"
         }
     ],
-    options: { x: -6, y: -6, scale: 0.25 }
+    options: { x: -6.2, y: -6, scale: 0.25 }
```

Comment:
What is this icon `X_LIGHT`?

Reply:
we call light colored shape of X_LIGHT.
![image](https://user-images.githubusercontent.com/63670637/91891608-f3cf1980-ec56-11ea-89a1-3e61434ff491.png)

Here is the link for all light colors present in carbon https://engineering.cerner.com/carbon-graphs/#/styles/shapes/standard-light